src/style.css

```diff
@@ -2,0 +1,92 @@
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir', Arial, sans-serif;
+}
+
+input {
+  all: unset;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: "Avenir" ;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+.search-box {
+  position: relative;
+  margin: 20px 0;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 4px;
+}
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here, you've added a margin with `margin: 20px 0;` which is fine, but ensure that the rest of the elements follow the same convention of either top or bottom margin for consistency and to avoid margin collapse issues.